setup.py

```diff
@@ -435,7 +436,8 @@ def setup_package():
         maintainer="NumPy Developers",
         maintainer_email="numpy-discussion@python.org",
         description=DOCLINES[0],
-        long_description="\n".join(DOCLINES[2:]),
+        long_description= Path("README.md").read_text(encoding="utf-8"),
```

Comment:
I think I would prefer to remove the docstring from this file, as its only real use is for `DOCLINES`.